components/sections/Footer/index.module.css

```diff
@@ -0,0 +1,38 @@
+.footer {
+  @apply flex
+    flex-col
+    items-center
+    gap-6
+    px-8
+    md:flex-row
+    md:justify-between;
+
+  .left {
+    @apply flex
+      gap-1;
```

Comment:
Need to check the mobile design for this element, doesn't match with Figma

Reply:
matched styling with Figma
![image](https://github.com/nodejs/nodejs.org/assets/19715103/a936ce8c-8243-4aa4-8d13-ca2884577c94)